algolia/algoliasearch-sfcc-b2c
# fix: reintroduce code of legacy jobs

To ensure a smooth upgrade, we want to keep the code associated to the recently introduced jobs (AlgoliaProductInventoryExport, AlgoliaProductPricesExport and AlgoliaProductsDeltaExport)

This PR reintroduce the code and step definitions that those jobs are relying on:
- `custom.sendChunkOrientedProductUpdates`
- `custom.algoliaSendDeltaExportProducts`
- `sendChunkOrientedProductUpdates.js`
- (sendDeltaExportProducts.js) was still present in the repo

Those files and steps have been copied from `master`.
The only changes is that they are tagged as deprecated, and I've shortened the steps descriptions so that they fit in the BM tooltip (see 757823f56e92d5461bdaf97573b7cb6d5d6e6c85)

### How to test

- Install the current cartridge version
- Switch to this branch and install the cartridge from here (including importing the jobs)

Jobs import should succeed, and 10 jobs are visible: the 5 legacy jobs and the 5 new.

---
SFCC-177
